gocwebtemplate-sample-jsp/src/main/webapp/samplecontents/footerlinkssamplecontent.jsp

```diff
@@ -1,7 +1,6 @@
-<h1>GoC Web Template Samples - Footer Links</h1>
-<p>The footer links consist of the Contact, News and About links located at the bottom of the page.  The environment you specify for your site determines which category and links are displayed.  The environment also determines which of the categories can be modified by the application.</p>
-<h2>Before Changing The Footer</h2>
-<p>Will you need a custom header/footer? Keep in mind this is different than removing or adding certain elements that are allowed as per the C&amp;IA specifications document. Any element that can be implemented using the C&amp;IA specifications document does not require a custom header/footer and is available in the default version of the CDTS. Currently it is advisable to not implement a custom header/footer unless you have permission to do so from your department, TBS or Principal Publisher. If you have permission to do so then follow the instructions for the "Application templates" in the menu on the right.</p>
+<h1>GoC Web Template Samples - Footer</h1>
+<p>Canada.ca is moving to a new global site footer as seen <a href="https://design.canada.ca/common-design-patterns/site-footer.html">here.</a></p>

```

Comment:
Good sample.  My only comment would be to keep it "timeless" so it stays relevant longer.  So instead of "Canada.ca is moving to..."   maybe something like "Canada.ca provides a global site footer as seen here..."  (or something better :) )

Reply:
done! (used what you said lol)